neuralop/layers/local_fno_block.py

```diff
@@ -19,107 +21,136 @@
+        if "tanh", use tanh
+    norm : Literal["ada_in", "group_norm", "instance_norm"], optional
+        Normalization layer to use, by default None
+    ada_in_features : int, optional
+        number of features for adaptive instance norm above, by default None
+    preactivation : bool, optional
+        whether to call forward pass with pre-activation, by default False
+        if True, call nonlinear activation and norm before Fourier convolution
+        if False, call activation and norms after Fourier convolutions
+    fno_skip : str, optional
+        module to use for FNO skip connections, by default "linear"
+        see layers.skip_connections for more details
+    channel_mlp_skip : str, optional
+        module to use for MLP skip connections, by default "soft-gating"
+        see layers.skip_connections for more details
+    SpectralConv Params
+    -------------------
+    separable : bool, optional
+        separable parameter for SpectralConv, by default False
+    factorization : str, optional
+        factorization parameter for SpectralConv, by default None
+    rank : float, optional
+        rank parameter for SpectralConv, by default 1.0
+    SpectralConv : BaseConv, optional
+        module to use for SpectralConv, by default SpectralConv
+    joint_factorization : bool, optional
+        whether to factorize all spectralConv weights as one tensor, by default False
+    fixed_rank_modes : bool, optional
+        fixed_rank_modes parameter for SpectralConv, by default False
+    implementation : str, optional
+        implementation parameter for SpectralConv, by default "factorized"
+    decomposition_kwargs : _type_, optional
+        kwargs for tensor decomposition in SpectralConv, by default dict()
+    fft_norm : str, optional
+        how to normalize discrete fast Fourier transform, by default "forward"
+        if "forward", normalize just the forward direction F(v(x)) by 1/n (number of total modes)
+    FiniteDifferenceConvolution Params
+    ----------------------------------
+    diff_layers : bool list, optional
+        Must be same length as n_layers, dictates whether to include a
+        differential kernel parallel connection at each layer
+    fin_diff_implementation : str in ['subtract_middle', 'subtract_all'], optional
+        Implementation type for FiniteDifferenceConvolution.
+        See differential_conv.py.
+    conv_padding_mode : str in ['periodic', 'circular', 'replicate', 'reflect', 'zeros'], optional
+        Padding mode for spatial convolution kernels.
+    default_in_shape : Tuple[int]
+        Default input shape on spatiotemporal dimensions. Repeated above for differential kernel.
+    fin_diff_kernel_size : odd int, optional
+        Conv kernel size for finite difference convolution.
+    mix_derivatives : bool, optional
+        Whether to mix derivatives across channels
+    EquidistantDiscreteContinuousConv2d Params
```

Comment:
This section won't render properly. I suggest moving to `Parameters` and `Other Parameters`

Reply:
Thanks! I updated it to follow the convention in `FNOBlocks`